ts/utils/testWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { Provider } from "react-redux";
+// import { Button } from "react-native";
+import { render } from "@testing-library/react-native";
+import {
+  createAppContainer,
+  NavigationContainer,
+  createStackNavigator
+} from "react-navigation";
+
+// Creates a simple wrapper for the render method which encloses the element to render in a store provider
+export const renderWithRedux = (
+  ui: any,
+  { initialState, store, ...renderOptions }: any = {}
```

Comment:
The use of `any` should be limited to when it is strictly necessary. Could you try to replace the any of this pr with the right types? Also for test utility function type safeness should be guaranteed (whenever possible) or could be really hard to find problems in case of types refactoring.

In this case, for example, `ui` could be `React.ReactElement`, `renderOptions` could be `RenderOptions`, etc.

Reply:
Sometimes I used `any` because props types don't have a common ancestor, so I can't write general code, but I'll try to be as strict with typing as possible

Reply:
@fabriziofff I have tried a solution to the abuse of `any` using `generics` and types you suggested. I've also changed the way I spy on modal events. I'd love to share/discuss the solution with you, when you have time. Feel free to ping me on Slack whenever you want